STTBot/events/app_mention.py

```diff
@@ -302,7 +302,15 @@ def _cmd_pin_remove(client, event_data, command, say):
     data_interface.remove_pin(permalink.channel, permalink.timestamp)
     return {"message": ":white_check_mark: Successfully removed pin"}
 
-
+def _STT_draft(client, event_data, command, say):
+    if len(command.args) == 0;
+        raise CommandError("No list provided")
+    else:
+        draft_order = command.args
+        random.shuffle(draft_order)
```

Comment:
You need to import random at the top of the file for this to work

Reply:
done